### DISCUSSION TITLE
Boletim Informativo - Janeiro, 2023

### DISCUSSION THREAD

**Discussion by tiidadavena** [apyb/comunidade#71] [Boletim]
# Boletim Informativo- Janeiro, 2023

Olá, pessoal! Esperamos que vocês estejam bem e que possamos nos encontrar virtualmente e presencialmente neste ano de 2023!

Segue mais uma edição do boletim informativo mensal da APyB levando até vocês as principais atividades realizadas durante o mês de janeiro de 2023.

### Acompanhamento de eventos
O ano 2023 promete ser muito agitado para pythonistas do Brasil! As organizações da PyNorte, Python Nordeste e Python Brasil já iniciaram suas atividades e estão preparando eventos incríveis!

Inclusive, se você tem interesse em colaborar na organização, a PyNorte está aceitando pessoas voluntárias. Acompanhe as redes dos eventos para saber mais detalhes e entrar em contato com a organização <3

Detalhes: [Link do servidor do Discord da organização do PyNorte](https://discord.gg/P9BAfKyZ)

### Associação à APyB
Você pode se associar à APyB em qualquer época do ano, como efetivo ou estudante. Neste ano em especial é bastante importante ser uma pessoa associada, pois durante a assembleia ordinária que ocorrerá na Python Brasil, teremos eleições para uma nova gestão.

Detalhes: [Link para associar-se](https://python.org.br/apyb/#associar)

## Outros tópicos
- Confira as demais [issues](https://github.com/apyb/tarefas/issues?q=is%3Aissue+is%3Aclosed+closed%3A2023-01-01..2023-01-31) fechadas no mês de janeiro.

[@apyb/diretoria](https://github.com/orgs/apyb/teams/diretoria) 👋

**Comment by jjpaulo2** [comment]
Já existe data definida para a Python Nordeste?

**Reply by tiidadavena** [reply]
Olá!
Ainda não, mas a data deverá ser divulgada em breve. Para mais infos, acesse: https://instagram.com/pythonnordeste?igshid=YmMyMTA2M2Y=

**Reply by rougeth** [reply]
@jjpaulo2 dá uma olhada no https://eventos.python.org.br, você pode incorporar o .ics na sua agenda e ficar por dentro dos eventos :)
